From: Blaise <BlaiseD@GMail.com>
Date: Sun, 23 Aug 2020 14:32:55 -0500
Subject: Disable the DIAL repeating discovery

This causes unnecessary SSDP network spam
FILE:Disable-the-DIAL-repeating-discovery.patch
---
diff --git a/chrome/browser/media/router/discovery/dial/dial_registry.cc b/chrome/browser/media/router/discovery/dial/dial_registry.cc
index 6fdc21949f..251e3ae2f2 100644
--- a/chrome/browser/media/router/discovery/dial/dial_registry.cc
+++ b/chrome/browser/media/router/discovery/dial/dial_registry.cc
@@ -192,10 +192,6 @@ void DialRegistry::StartPeriodicDiscovery() {
 
   dial_ = CreateDialService();
   dial_->AddObserver(this);
-  DoDiscovery();
-  repeating_timer_.reset(new base::RepeatingTimer());
-  repeating_timer_->Start(FROM_HERE, refresh_interval_delta_, this,
-                          &DialRegistry::DoDiscovery);
 }
 
 void DialRegistry::DoDiscovery() {
@@ -209,8 +205,6 @@ void DialRegistry::StopPeriodicDiscovery() {
   if (!dial_)
     return;
 
-  repeating_timer_->Stop();
-  repeating_timer_.reset();
   dial_->RemoveObserver(this);
   ClearDialService();
 }
